classes/class-wc-connect-logger.php

```diff
@@ -2,16 +2,21 @@
 
 if ( ! class_exists( 'WC_Connect_Logger' ) ) {
 
+	define( 'WOOCOMMERCE_CONNECT_DEBUG_LOGGING_ENABLED_OPTION', 'wcc_debug_logging_enabled' );
```

Comment:
Because the constant is used in multiple places - didn't want to risk a change not hitting all the instances of the option name